### DISCUSSION TITLE
Letsencrypt staging cert

### DISCUSSION THREAD

**Discussion by techserver5110** [nextcloud/all-in-one#623] [Questions] [answered]
Hi all,

I've had the nextcloud aio basic set up running successfully earlier today.

Unfortunately, due to experimenting on the reverse proxy server, I've had to rebuild 5 times and thus have depleted the full limit of letsencrypt production certs from the letsencrypt issuer.

Is there a way to switch the issuer while in development to "staging" (i.e. to feed from https://acme-staging-v02.api.letsencrypt.org rather than from  https://acme-v02.api.letsencrypt.org)

Thanks.

Paul

**Comment by szaimen** [comment, marked as answer]
> Is there a way to switch the issuer while in development to "staging" (i.e. to feed from https://acme-staging-v02.api.letsencrypt.org rather than from https://acme-v02.api.letsencrypt.org)

No, sorry, there is no way to do that. If you want to run it behind a reverse proxy, you should follow the reverse proxy documentation which will not issue an certificate automatically...

**Reply by techserver5110** [reply]
Thanks for the quick response
